### PR TITLE
app base new structure

### PR DESCRIPTION
**Pull request by faisal3413**
completes app structure

closes #16

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,57 +1,36 @@
...
+
+const store = configureStore();
+registerScreens(store, Provider);
 
-const instructions = Platform.select({
-  ios: 'Press Cmd+R to reload,\n' +
-    'Cmd+D or shake for dev menu',
-  android: 'Double tap R on your keyboard to reload,\n' +
-    'Shake or press menu button for dev menu',
+const navigatorStyle = {
```
Comment (florianlenz) [inline App.js]: Shouldn't they be in an separate stylesheet or smth like that?

File: src/global/Colors.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+/*!
+ * Theme colours
+ */
+module.exports = {
+  Red: '#FF5252',
+  Pink: '#FF4081',
```
Comment (florianlenz) [inline src/global/Colors.js]: Are the color's from our styleguide? I don't remember that we used "Pink".

**Comment by florianlenz** [comment]
@faisal3413 Can you updated the readme with the new folder's / structure?

File: src/global/Colors.js (hunk 2 above)
Comment (faisal3413) [reply src/global/Colors.js]: I just used a common colors file. I will remove which is not required.

File: App.js (hunk 1 above)
Comment (faisal3413) [reply App.js]: Okay, I will revise.

**Comment by florianlenz** [comment]
I am closing this. There went something wrong with our merge (sorry for that). I created a new branch base from commit  #c7c6acd9d5074a1895d3e568d01f4ac40f0394f1 The branch name is feature/app_structure

**Pull request closed without merging**